etc/ui-config/mh_default_org/paella/config.json

```diff
@@ -119,6 +119,11 @@
     },
     "//**** For a list of available plugins and configuration, go to": "https://github.com/polimediaupv/paella/blob/master/doc/plugins.md",
     "list": {
+      "es.upv.paella.opencast.loader": {
+        "flavors": [
+          ["*", "*"]
```

Comment:
Why Do you choose this notation instead of "*/*" to be consistent with other workflow configuration files?
I think it is more readable 
```
flavors:["*/*"]
```
than
```
flavors:[["*", "*"]]
```

What do you think?

Reply:
It's essentially the same and the array variant is less to parse.